src/components/Projects.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import * as S from "./Projects.styles";
+
+interface projectProps {
+  image?: string;
+  github: string;
+  description: string;
+  name: string;
+}
+function Project(props: projectProps) {
+  const [descriptionIsVisible, setDescriptionIsVisible] = useState(false);
+  const [nameIsVisible, setNameIsVisible] = useState(false);
+
+  function mouseLeave() {
+    setDescriptionIsVisible(false);
+    setNameIsVisible(false);
+  }
+  function mouseEnter() {
+    setDescriptionIsVisible(true);
+    setNameIsVisible(true);
+  }
```

Comment:
Instead of hardcoding the values in 2 separate functions, I would suggest having one toggle function (and in arrow notation to be consistent with the rest of the codebase) like so:

```
const toggleProjectDetails = () => {
    setDescriptionIsVisible(!descriptionIsVisible);
    setNameIsVisible(!nameIsVisible);
}
```

And call this on both mouse enter and mouse leave